### ISSUE TITLE
Accept string as `content-transition` value

### ISSUE THREAD

**Issue by kiaking** [ktsn/vue-thin-modal#18]
Is it possible to make `content-transition` to accept string as well as object? Most of the time I just want to change the name of the transition, and I thought it might be nice if `content-transition` could accept both string and object 😃 

```vue
<template>
  <modal content-transition="fade">
    <div class="my-modal">...</div>
  </modal>
</template>

<style scoped>
.my-modal.fade-enter,
.my-modal.fade-leave-to {
  opacity: 0;
}
</style>
```

**Comment by ktsn** [comment]
Sounds good 👍

**Comment by kiaking** [comment]
Great! Will you implement this? I could make PR too (I hope I can 😀) so please let me know if I should 🚀

**Comment by ktsn** [comment]
It's published in v1.2.0. 🚀

**Comment by kiaking** [comment]
Yay! Thank you so much!
